Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift

```diff
@@ -50,28 +50,28 @@ final class ChatMessageImageView: UIView {
             addGestureRecognizer(tapGesture)
         }
 
-		guard let imageURL = attachment.fullImageURL() else {
-			return
-		}
-		var placeholderImage: UIImage?
+        guard let imageURL = attachment.fullImageURL() else {
+            return
+        }
+        var placeholderImage: UIImage?
         if imageURL.absoluteString.starts(with: "http://") {
-        	isLoadable = false
-			labelTitle.text = localized("alert.insecure_image.title") + " (" + attachment.title + ")"
-			placeholderImage = UIImage(named: "Insecure Image")
-		} else {
-			labelTitle.text = attachment.title
-		}
+            isLoadable = false
+            labelTitle.text = attachment.title + " (" + localized("alert.insecure_image.title") + ")"
```

Comment:
@cardoso I just reordered the title/alert so the warning is now after the title and may fade out, but the image+alert should be enough